src/components/RepoList.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div id="repoList">
+    <h1>{{ userName }}</h1>
+    <div id="totalCount">
+      <p>{{ totalRepoCount }} repositories</p>
+      <p>|</p>
+      <p>{{ totalRepoStarCount }} stars</p>
```

Comment:
p태그마다 margin을 주기위한 의도였다면 상위 div를 flex로 주시고 Template Literal로 감싸줘도 좋을 것 같습니다 👍 
`#totalCount {
      display: flex;
      justify-content: space-between; //space-around;
}`
 
```suggestion
      <p>${ totalRepoCount } repositories | ${ totalRepoStarCount } star</p>
```

Reply:
헛 그러네요 ㅎㅎ 왜 저땐 저생각을 못했을까요...? 감사합니다 ㅎㅎ